templates/20years/index.html

```diff
@@ -773,32 +837,86 @@ <h2>
+          </p>
+          <hr class="p-rule--muted" />
+          <div class="p-heading-icon--small">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img"
+                   src="https://assets.ubuntu.com/v1/965ec1de-LinkedIn%20Logo%20-%20Updated%202.png"
+                   alt="LinkedIn logo" />
+              <p class="p-heading-icon__title">
+                <a href="https://www.linkedin.com/posts/keegan-turner-20716615b_20-years-of-ubuntu-activity-7245407163685191680-qR7S?utm_source=share&utm_medium=member_desktop">See post&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+          </div>
+        </div>
+        <div class="p-card--social-media">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img"
+                   src="https://assets.ubuntu.com/v1/523f779c-Shannon%20smith.jpeg"
+                   alt="Shannon Smith photo" />
+              <p class="p-heading-icon__title">
+                <strong><small>LinkedIn post by Shannon Smith</small></strong>
+              </p>
+            </div>
+          </div>
+          <p>20 years of Ubuntu , the OS that we all love ❤️</p>
+          <hr class="p-rule--muted" />
+          <div class="p-heading-icon--small">
+            <div class="p-heading-icon__header">
+              <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/965ec1de-LinkedIn%20Logo%20-%20Updated%202.png" alt="LinkedIn logo" />
+              <p class="p-heading-icon__title">
+                <a href="https://www.linkedin.com/posts/shannonandriansmith_ubuntults-opensource-activity-7191994640412151808-KUQW?utm_source=share&utm_medium=member_desktop">See post&nbsp;&rsaquo;</a>
+              </p>
+            </div>
+          </div>
+        </div>
       </div>
       <div class="col-3 col-medium-2">
         <div class="p-card--social-media">
           <div class="p-heading-icon">
             <div class="p-heading-icon__header">
-              <img class="p-heading-icon__img"
-                   src="https://assets.ubuntu.com/v1/4334e00b-@missingfaktor.png"
-                   alt="" />
               <p class="p-heading-icon__title">
-                <strong><small>Tweet by
-                @missingfaktor</small></strong>
+                <strong><small>LinkedIn post by Mohsen Mostafa Jokar</small></strong>
               </p>
             </div>
           </div>
-          <p>Happy 20 years, Ubuntu! 🎉</p>
           <p>
-            I still fondly look back on my first installation with 7.04 Feisty Fawn. It's amazing to see how far Ubuntu has come since then. Here's to many more years! 🧡🤎💜
+            20 years of Ubuntu: Canonical celebrates with upcoming 24.10 - and it's outstanding as usual <br/> #Linux hashtag #Ubuntu hashtag #Desktop hashtag #Canonical
```

Comment:
```suggestion
            20 years of Ubuntu: Canonical celebrates with upcoming 24.10 - and it's outstanding as usual <br/> &#35;Linux &#35;Ubuntu &#35;Desktop &#35;Canonical
```
Could use encodings here (and everywhere else)